docs/pages/api.rst

```diff
@@ -1171,6 +1174,26 @@ Signature: ``Collection::intersperse($element, int $every = 1, int $startAt = 0)
         var_dump($item); // 'x', 'a', 'x', 'b', 'x', 'c'
     }
 
+isEmpty
+~~~~~~~
+
+Check if a collection has any elements inside.
+
+Interface: `IsEmptyable`_
+
+Signature: ``Collection::isEmpty();``
+
+.. code-block:: php
+
+    Collection::fromIterable(range('a', 'c'))
```

Comment:
One minor remark. How about moving the example in its own file like we did for others ?

Reply:
I just did it so it's ok.